mne/commands/tests/test_commands.py

```diff
@@ -233,13 +238,13 @@ def test_watershed_bem(tmpdir):
 @pytest.mark.ultraslowtest
 @requires_freesurfer
 @sample.requires_sample_data
-def test_flash_bem(tmpdir):
+def test_flash_bem():
     """Test mne flash_bem."""
     check_usage(mne_flash_bem, force_help=True)
     # Using the sample dataset
     subjects_dir = op.join(sample.data_path(download=False), 'subjects')
     # Copy necessary files to tempdir
-    tempdir = str(tmpdir)
+    tempdir = _TempDir()
```

Comment:
We want almost everything from this commit. Feel free to push a new PR but we shouldn't lose them

Reply:
no prob. I'll get it back and just ping you to make sure I did not forget anything important.

Reply:
Actually I made a PR https://github.com/mne-tools/mne-python/pull/6192 feel free to push in it.